ToDoApp/ToDoApp/ToDoService.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ToDoDetail.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-11-29.
+//
+
+import Foundation
+
+enum TaskError: Error {
+    case unavailableTask
+}
+
+struct Task {
+    var id: String = ""
+    var name: String = ""
+    var deadline: Date = Date.now
+    var status: Bool = false
+    init(name: String, deadline: Date?) {
```

Comment:
suggestion: Swift automatically synthesizes initializers for structs, so I most cases you don't need to write it yourself